xen-ocaml/install.sh

```diff
@@ -23,7 +23,6 @@ mkdir -p $odir/mirage-xen-ocaml
 #We dont install the bytecode version yet
 #cd ocaml-src/byterun && make install LIBDIR="${pwd}/obj" BINDIR="${pwd}/obj"
 cp ocaml-src/$ASMRUN_FOLDER/libasmrun.a $odir/mirage-xen-ocaml/libxenasmrun.a
-ln -s $odir/mirage-xen-ocaml/libxenasmrun.a $odir/ocaml/libasmrunxen.a
```

Comment:
Could you add a local symlink to rename the file ? Something like
`ln -s $odir/mirage-xen-ocaml/libxenasmrun.a $odir/mirage-xen-ocaml/libasmrun.a`
Otherwise the `-I` trick won't work as OCaml looks specifically for `libasmrun.a`.

Reply:
sure, what about `cp ocaml-src/$ASMRUN_FOLDER/libasmrun.a $odir/mirage-xen-ocaml/`? ~~> not installing any `libxenasmrun.a`? (I'm not entirely sure which files are needed and used by which tools - could we as well use `-I` and `-runtime-variant`, and leave `libxenasmrun.a` as is?)

Reply:
We can either name it `libasmrun.a` or `libasmrunxen.a` (with `-runtime-variant xen`). The file is only used in the final link step performed by mirage.